src/components/settings/SettingsLMSTab/LMSConfigs/Canvas/CanvasConfig.tsx

```diff
@@ -0,0 +1,126 @@
+import { SyntheticEvent } from "react";
+import isEmpty from "lodash/isEmpty";
+
+import handleErrors from "../../../utils";
+import LmsApiService from "../../../../../data/services/LmsApiService";
+import { snakeCaseDict } from "../../../../../utils";
```

Comment:
nit: `@edx/frontend-platform/utils` also exports a `snakeCaseObject` utility function, similar to its `camelCaseObject` function.